misk/src/test/kotlin/misk/web/interceptors/LogRateLimiterTest.kt

```diff
@@ -23,23 +22,23 @@ class LogRateLimiterTest {
 
   @Test
   fun tryAcquire() {
-    val bucketId = LogBucketId("caller", TestAction::class, false)
+    val bucketId = LogBucketId("TestAction", false)
     assertThat(
-      logRateLimiter.tryAcquire(bucketId)).isTrue()
+      logRateLimiter.tryAcquire(bucketId, 2L)).isTrue()
     assertThat(
-      logRateLimiter.tryAcquire(bucketId)).isTrue()
+      logRateLimiter.tryAcquire(bucketId, 2L)).isTrue()
     assertThat(
-      logRateLimiter.tryAcquire(bucketId)).isFalse()
+      logRateLimiter.tryAcquire(bucketId, 2L)).isFalse()
 
     // Wait 1 second
     fakeTicker.sleepMs(1000L)
-    assertThat(logRateLimiter.tryAcquire(bucketId)).isTrue()
+    assertThat(logRateLimiter.tryAcquire(bucketId, 2L)).isTrue()
   }
 
   @Test
   fun noRateLimiting() {
     assertThat(
-      logRateLimiter.tryAcquire(LogBucketId("caller", NoLogRateLimitingTestAction::class, false))
+      logRateLimiter.tryAcquire(LogBucketId("NoLogRateLimitingTestAction", false), 0L)
     ).isTrue()
```

Comment:
Calling this a few times in succession seems like it would be a better test.

Reply:
d'oh